kie-soup-dataset/pom.xml

```diff
@@ -40,7 +40,7 @@
     <module>kie-soup-dataset-sql</module>
     <module>kie-soup-dataset-sql-tests</module>
     <module>kie-soup-dataset-csv</module>
-    <module>kie-soup-dataset-elasticsearch</module>
+    <!--module>kie-soup-dataset-elasticsearch</module-->
```

Comment:
Shouldn't it be removed instead of only commenting?

Reply:
yes, even i intentionaly kept it for later maybe any refactoring or whatever ;) the module itself is not removed from GIT

Reply:
i created https://issues.redhat.com/browse/AF-2944 for this final task.